templates/reconciliation.js

```diff
@@ -35,6 +35,48 @@ var vote = function vote($choice){
   nextPairing($pairing);
 }
 
+// This function looks through allVotes() to see if there are any IDs
+// matched with multiple UUIDs or (conversely) any UUIDs matched with
+// multiple IDs. If so, it returns HTML that warns about these
```

Comment:
It would be good to split this in two, so that the HTML generation is separate from finding the problem cases. Simply showing the HTML output is a good first step, but to really be useful we'll need to offer an interface to _do_ something with these (even if just to un-match them for later re-matching), at which point we'll need the data, not just the text.